### PR TITLE
feat: add rembg

### PR DESCRIPTION
**Pull request by turbo1912**

### CODE REVIEW

File: src/data/modelMetadata.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  apiEndpoint: string;
+  pythonExampleEndpoint: string;
+  javascriptExampleEndpoint: string;
+  code: string;
+};
+
+const regmbModel: Model = {
+  id: "rembg",
+  name: "Rembg",
+  apiEndpoint: "https://103961668-rembg.gateway.alpha.fal.ai",
```
Comment (drochetti) [inline src/data/modelMetadata.ts:13]: move this to public ENV var?

File: src/pages/api/rembg.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  axios
+    .post(`${REMBG_URL}/remove`, formData)
+    .then(async (res) => {
+      if (res.status == 200) {
+        await incrementImageCount({ by: 1 });
+
+        response.status(res.status).send({ imageUrl: res.data });
+      }
+    })
+    .catch((error) => console.error(error));
```
Comment (drochetti) [inline src/pages/api/rembg.ts:38]: just curious here, did fetch not work for this request? I remember we had issues with different clients, so trying to remember if this was the case.

**Review by drochetti — APPROVED** [review]

File: src/pages/api/rembg.ts (hunk 2 above)
Comment (turbo1912) [reply src/pages/api/rembg.ts:38]: actually after we made the change to allow the transfer-encoding that this request produces, node-fetch will work

File: src/data/modelMetadata.ts (hunk 1 above)
Comment (turbo1912) [reply src/data/modelMetadata.ts:13]: the whole thing or just the api endpoint part?

**Pull request merged**